tests/test-scribe.c

```diff
@@ -542,8 +542,9 @@ main (int argc, char *argv[])
   TestData good_signature_truncated_gz = {
       .image_path = trunc_gz_path,
       .signature_path = trunc_gz_sig_path,
-      .error_domain = G_IO_ERROR,
-      .error_code = G_IO_ERROR_PARTIAL_INPUT,
+      /* TODO: report a kinder error than "child process exited with code 1" */
```

Comment:
TODO comment here. (Ignore this comment if it’s addressed in a later commit. I’m reviewing the commits sequentially.)

Reply:
And another below.

Reply:
I'd expect to fix this at the same time as the pervasive use of `0` error codes. That said, I guess it is worth fixing this because, when the image file is truncated, there's a race between GPG and the decompressor to find out who will notice the error first…